OrbitCore/FunctionFramepointerValidator.cpp

```diff
@@ -71,9 +83,10 @@ bool FunctionFramepointerValidator::ValidatePrologue() {
 // all are correct. However, we would not expect a compiler to produce this,
 // and for hand written assembly, we accept wrong unwinding results.
 bool FunctionFramepointerValidator::ValidateEpilogue() {
-  for (size_t i = 0; i < instructions_count_ - 1; i++) {
+  for (size_t i = 0; i < instructions_count_ - 2; i++) {
     // "leave" is equivalent to "mov esp, ebp" "pop ebp"
```

Comment:
not always - apparently it depends on nesting attribute. it could do more if I understand correctly

Reply:
As discussed offline, we would still accept this here.